pkg/store/filesystem/io.go

```diff
@@ -0,0 +1,42 @@
+package store
```

Comment:
could you rename `io.go` to `main.go`? I would do the same on the occis-accounts service, for consistency.

Reply:
But its not main, it is only the io, i.e. marshaling and unmarshaling with proper file locks.